adyen.php

```diff
@@ -547,19 +549,64 @@ public function hookOrderConfirmation($params)
      */
     public function hookPaymentOptions($params)
     {
+        $amount = $this->context->cart->getOrderTotal();
+        $currency = $this->context->currency->iso_code;
+        $address = new Address($this->context->cart->id_address_invoice);
+        $isoAddress = Country::getIsoById($address->id_country);
+        $shopperReference = $this->context->cart->id_customer;
+        $shopperLocale = $this->context->language->iso_code;
```

Comment:
below we use $this->helper_data->getLocale($this->context) should we use it here as well ?